packages/kas/src/nodes.ts

```diff
@@ -3553,13 +3615,15 @@ export const parse = function (input, options) {
         var expr = parser.parse(input).completeParse();
         return {parsed: true, expr: expr};
     } catch (e) {
-        return {parsed: false, error: e.message};
+        return {parsed: false, error: (e as Error).message};
```

Comment:
Is this possible? I feel like probably not or else I imagine you would have done this.

```TS
    } catch (e: Error) {
        return {parsed: false, error: e.message;
    }
```